conf.d/charts.d.conf

```diff
@@ -56,8 +56,3 @@
 # sensors=force
 # squid=force
 # tomcat=force
-
-# OLD MODULES THAT ARE NOW SERVED BY NETDATA DAEMON
-# cpu_apps=force
-# mem_apps=force
-# load_average=force
```

Comment:
Please don't remove these lines.
The next update of netdata will enable them, since they are already installed.